.github/CODEOWNERS

```
@@ -75,3 +75,13 @@ subprojects/docs/src/docs/userguide/dep-man/
 subprojects/docs/src/docs/userguide/jvm/                                          @gradle/bt-jvm
 subprojects/docs/src/docs/userguide/reference/directory_layout.adoc               @gradle/bt-jvm
 subprojects/docs/src/docs/userguide/troubleshooting/version_catalog_problems.adoc @gradle/bt-jvm
+
+# Dev-prod team
+.teamcity/                                  @gradle/bt-developer-productivity
+.github/                                    @gradle/bt-developer-productivity
+gradle/shared-with-buildSrc                 @gradle/bt-developer-productivity
```

```suggestion
gradle/shared-with-buildSrc/                @gradle/bt-developer-productivity
```

Is there a `/` missing at the end here? What is the consequence of leaving it off?

---

No idea, but let's add it.